VMEncryption/main/handle.py

```diff
@@ -1850,7 +1963,12 @@ def daemon_decrypt():
                           code=CommonVariables.encryption_failed,
                           message='Decryption failed for {0}'.format(failed_item))
         else:
-            encryption_config.clear_config()
+            encryption_status = json.loads(disk_util.get_encryption_status())
```

Comment:
check that file exists before loading, and that encryption_status is not None before use

Reply:
Will do.

Reply:
Not seeing these checks yet - is this covered now, or still pending?